tests/tests.c

```diff
@@ -1538,7 +1538,8 @@ unsigned sirtest_thread(void* arg) {
 #endif
     }
 
-    printf("\thi, i'm thread id %d, logging to: '%s'...\n", threadid, my_args->log_file);
+    printf("\thi, i'm thread id %lld, logging to: '%s'...\n",
```

Comment:
Sorry, I don't wish to do this. It makes no sense to cast on every platform for one that is different; we should cast that to int IMHO.

Reply:
Will cast.